src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationCellModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import Foundation
+import UIKit
+
+struct AppInformationCellModel {
```

Comment:
According to the search we have `AppInformationLegalCell` and `AppInformationDynamicCell`. We should consider good 'ol inheritance and put this model as inline struct into a common `AppInformationCell`. Less files, better code.

I can help with this!